### ISSUE TITLE
Guid.TryParse throws on null

### ISSUE THREAD

**Issue by Suchiman** [bridgedotnet/Bridge#3195]
Guid.TryParse throws on null input argument.

### Steps To Reproduce

https://deck.net/3ff65c70c72d5610ae3cae6dd3c3304d

```csharp
public class Program
{
    public static void Main()
    {
        Guid t;
        bool result = Guid.TryParse(null, out t);
        
        Console.WriteLine(result);
    }
}
```

### Expected Result
[returns false](https://sharplab.io/#v2:C4LgTgrgdgPgAgBgARwIwG4CwAoHcDMKATCqgOw4DeOStKhaAbCgCykAcAFAJQ13XY6QpAHEIASwAmSYFkHDaAIwD2ygDZIwAUwDOENcCQBeURMkA6ACpgAngAUAhmB1bOUfWoA0SZREPBuOQVaPgU0AE5ObT0DQNCkAF8cBKA==)

### Actual Result
[throws exception](https://deck.net/3ff65c70c72d5610ae3cae6dd3c3304d)

**Comment by geoffreymcgill** [comment]
Hi @Suchiman. Thanks for letting us know about this defect. we investigate and try to fix right away.

**Comment by fabriciomurta** [comment]
The associated pull request has just been merged in. The fix will be available to Bridge 16.4.0.
